jaybosamiya/nautilus-interactive-typeahead-debian
# Fix nautilus crashes

This fixes nautilus crashing whenever something is typed in the search box that yields no results and either `[Enter]` or `[Shift]+[Enter]` is pressed.

Steps to reproduce:

* prepare the following directory structure:
    ```
    testdirectory/
    ├── a/
    ├── b/
    └── c/

    3 directories, 0 files
    ```
* navigate into `testdirectory` with nautilus
* press the letter `d` on the keyboard, the typeahead prompt appears showing "d"
* hit `[Enter]` or  `[Shift]+[Enter]`
* nautilus crashes


I hacked the necessary fixes into the patch file using `editdiff`. This somehow got rid of the `index ...` lines in the patch file. Feel free to reintegrate those lines if you see a use for them.

**The packages within the `debs/` directory of this repository will also need a rebuild, I didn't include them in this Pull Request due to security reasons.**
